schema/bancor/view_smart_tokens.sql

```diff
@@ -0,0 +1,274 @@
+CREATE OR REPLACE VIEW bancor.view_smart_tokens (symbol, contract_address, decimals) AS VALUES
```

Comment:
Are there no events when smart-tokens are added, such that we could compute this dynamically?

Reply:
These are created in `BancorConverterRegistry_evt_SmartTokenAdded`. Problem, event doesn't contain `symbol` and `decimals`. If you have an idea how to fix this – glad to help.

For ref:
```
SELECT *
FROM bancor."BancorConverterRegistry_evt_SmartTokenAdded";
```

Reply:
Got it! Yeah, for now this is OK!